### DISCUSSION TITLE
How can I compile Arduino code with Vs Code?

### DISCUSSION THREAD

**Discussion by Junaid-Q** [microsoft/vscode-discussions#317] [Extension Development QnA] [answered]
Hi, I am struggling with the Arduino code using Vs Code IDE; however, there is a path error that I couldn't tackle. Can someone please let me know what error I repeat? 
I have already installed Arduino IDE on both PC and Vs Code extension. 

![image](https://user-images.githubusercontent.com/12583457/207391599-ea260405-50e4-4ff2-ab15-8291734da436.png)

Thanks

**Comment by gjsjohnmurray** [comment, marked as answer]
I think the right place to ask for help is with the extension authors.
